src/erlcloud_ses.erl

```diff
@@ -132,6 +145,147 @@ delete_identity(Identity, Config) ->
         {error, Reason} -> {error, Reason}
     end.
 
+%%------------------------------------------------------------------------------
+%% @doc
+%% Custom Verification Templates
+%% SES API:
+%% [https://docs.aws.amazon.com/ses/latest/APIReference/API_GetCustomVerificationEmailTemplate.html]
+%%
+%% Template attributes:
+%%   { template_name , string() }
+%%   { from_email_address , string() }
+%%   { template_subject , string() }
+%%   { template_content , string() }     -- please see notes in API Guide on what is allowed
+%%   { success_redirect_url , string() }
+%%   { failure_redirect_url , string() }
+%%
+%% On template creation, all attributes are mandatory.
+%% On template updates, only include the attributes you need to modify
+%% When listing templates, all attributes are included except template_content, use the get function to retrieve it
+%%
+%%  create_custom_verification_email_template
+%%  update_custom_verification_email_template
+%%  send_custom_verification_email
+%%  delete_custom_verification_email_template
+%%  get_custom_verification_email_template
+%%  list_custom_verification_email_templates
+%%
+%% @end
+%%------------------------------------------------------------------------------
+
+-type custom_template_attribute_names() :: template_name | from_email_address | template_subject | template_content | success_redirect_url | failure_redirect_url .
+-type custom_template_attributes() :: [ { custom_template_attribute_names() , string() } ].
+
+-type create_custom_verification_email_template_result() :: ok | { error , term() }.
+
+-spec create_custom_verification_email_template( string() , string() ,string() ,string() ,string() ,string() ) -> create_custom_verification_email_template_result().
```

Comment:
a couple of other style nits:

* the spacing between args, commas, and braces here (and in a number of other places in the code) is inconsistent to itself and the rest of the module. Some of the lines are also really, really wide. I prefer if you stick with
  ```erlang
  -spec my_fun(string(), string(), string) -> result().
  my_fun(Arg1, Arg2, Arg3) ->
      MyProplist = [{v1, Arg1},
                    {v2, Arg2},
                    {v3, Arg3}],
      do_stuff(MyProplist).
  
  
  -spec my_really_long_function_name(string(), string(), string(), aws_config()) ->
          my_really_long_function_name_result().
  ```
* the ordering of these functions in the module is inconsistent to the others here. It would be better if they were placed in alphabetical order into the module, consistent with how the others have been contributed.

Reply:
Agreed. Done in next check in.